### DISCUSSION TITLE
No more features until release. It's bugfix time

### DISCUSSION THREAD

**Discussion by PavelZinchenko** [PavelZinchenko/event-horizon-frontier-builds#165] [Announcements]
No more features until release. It's bugfix time

**Comment by Tree3324417** [comment]
>No more features until release

Are you really sure, with the mass of recent additions, especially on the pilot side?
Please, we are fine with features, but do what you say and focus on bugfixes, or at least do the enhancement suggestions relating to grinding as a first priority (these are of more importance as of now, in my opinion)

**Reply by PavelZinchenko** [reply]
Sorry guys, I got carried away. Fixing bugs is so boring, while making bugs is fun. Alright, no more features. I mean it.

**Reply by Tree3324417** [reply]
Thanks. I get it with creating bugs being fun, but a focus on resolving existing problems is needed too, especially when you say that you will do it 👍

**Comment by TaipionsSalemClient** [comment]
Thank You! :-)

**Comment by MakStarOne** [comment]
![GFDmE4HIy5o.jpg](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/160492285/72f3cd4f-d2d1-4037-ae43-56896b395787)
